Slack.Automation/Promact.Erp.DomainModel/Migrations/Configuration.cs

```diff
@@ -25,10 +25,10 @@ protected override void Seed(Promact.Erp.DomainModel.Context.PromactErpContext c
                 Question thirdQuestionTaskMail = new Question() { CreatedOn = DateTime.UtcNow, OrderNumber = 3, QuestionStatement = "What is the Status of this task?", Type = 2 };
                 Question fourthQuestionTaskMail = new Question() { CreatedOn = DateTime.UtcNow, OrderNumber = 4, QuestionStatement = "Mention if any Comment/Roadblock.", Type = 2 };
                 Question fifthQuestionTaskMail = new Question() { CreatedOn = DateTime.UtcNow, OrderNumber = 5, QuestionStatement = "Do you want to send your task mail?", Type = 2 };
-                Question sixthQuestionTaskMail = new Question() { CreatedOn = DateTime.UtcNow, OrderNumber = 6, QuestionStatement = "Are you sure to send mail? After sending email you won't be able to add any tak for today.", Type = 2 };
+                Question sixthQuestionTaskMail = new Question() { CreatedOn = DateTime.UtcNow, OrderNumber = 6, QuestionStatement = "Are you sure to send mail? After sending an email you won't be able to add any task for today.", Type = 2 };
```

Comment:
Questions statement are now updated as normal business English. Please review it